MerlinAU.sh

```diff
@@ -1801,7 +1811,7 @@ _toggle_beta_updates_() {
 
     if [ "$currentSetting" = "ENABLED" ]; then
         printf "${REDct}*WARNING*:${NOct}\n"
-		printf "Disabling updates from beta F/Ws to release F/Ws may prevent access to the latest features and fixes.\n"
+		printf Disabling updates from beta to release firmware may limit access to new features and fixes.\n"
```

Comment:
The "**printf**" statement in line **1814** is missing the opening double quotes before the word "Disabling"

Reply:
> The "**printf**" statement in line **1814** is missing the opening double quotes before the word "Disabling"

Good catch, fixing it now.